packages/interface/src/components/fullscreen-mode/test/index.js

```diff
@@ -1,76 +1,55 @@
 /**
  * External dependencies
  */
-import { act, create } from 'react-test-renderer';
+import { render } from '@testing-library/react';
 /**
  * Internal dependencies
  */
 import FullscreenMode from '..';
 
 describe( 'FullscreenMode', () => {
 	it( 'fullscreen mode to be added to document body when active', () => {
-		act( () => {
-			create( <FullscreenMode isActive={ true } /> );
-		} );
-		expect( document.body.classList.contains( 'is-fullscreen-mode' ) ).toBe(
-			true
-		);
+		const { baseElement } = render( <FullscreenMode isActive={ true } /> );
+
+		expect( baseElement ).toHaveClass( 'is-fullscreen-mode' );
 	} );
 
 	it( 'fullscreen mode not to be added to document body when active', () => {
-		act( () => {
-			create( <FullscreenMode isActive={ false } /> );
-		} );
-		expect( document.body.classList.contains( 'is-fullscreen-mode' ) ).toBe(
-			false
-		);
+		const { baseElement } = render( <FullscreenMode isActive={ false } /> );
+
+		expect( baseElement ).not.toHaveClass( 'is-fullscreen-mode' );
 	} );
 
 	it( 'sticky-menu to be removed from the body class if present', () => {
 		document.body.classList.add( 'sticky-menu' );
-		act( () => {
-			create( <FullscreenMode isActive={ false } /> );
-		} );
-		expect( document.body.classList.contains( 'sticky-menu' ) ).toBe(
-			false
-		);
+
+		const { baseElement } = render( <FullscreenMode isActive={ false } /> );
+
+		expect( baseElement ).not.toHaveClass( 'sticky-menu' );
 	} );
 
 	it( 'sticky-menu to be restored when component unmounted and originally present', () => {
 		document.body.classList.add( 'sticky-menu' );
-		let mode;
-		act( () => {
-			mode = create( <FullscreenMode isActive={ false } /> );
-		} );
-		act( () => {
-			mode.unmount();
-		} );
-		expect( document.body.classList.contains( 'sticky-menu' ) ).toBe(
-			true
+		const { baseElement, unmount } = render(
+			<FullscreenMode isActive={ false } />
 		);
+
+		unmount();
```

Comment:
Instead of calling `unmount`, does it make sense to just `rerender()` some `<div />` so the original component is no longer in the component tree, which will ultimately trigger an unmount?

Since this is part of the API it could be the most explicit way to express what we're aiming to test, though, so feel free to keep it as-is.

Reply:
The `unmount` makes sense to me when testing effect cleanup.